README.md

```diff
@@ -3,90 +3,121 @@ GOV.UK Frontend ·
 [![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
 =====================
 
-## What is it?
+:rotating_light: **GOV.UK Frontend is in private beta and these instructions are subject to change. If you want to know more about how you can use GOV.UK Frontend, please get in touch with the [Design System team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) at GDS.** :rotating_light:
 
-A single package containing everything needed to start building a GOV.UK service.
+## What is GOV.UK Frontend?
 
-:rotating_light: **This project is in development and these instructions are likely to change.** :rotating_light:
+GOV.UK Frontend contains the code you need to start building a user interface for government platforms and services. 
 
-Guidance for building a GOV.UK service - [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/).
+## Before using GOV.UK Frontend
 
-Want to help us build GOV.UK Frontend? See [contributing](#contributing)
+You should start by familiarising yourself with the [GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/), which contains guidance and examples of components, styles and patterns you can use to design  government platforms and services. 
+
+## How to use GOV.UK Frontend 
+
+There are 2 ways to use GOV.UK Frontend:
+* we recommend using <a href="#option1">option 1</a>, which requires using node package manager (NPM) 
+* you can also use <a href="#option2">option 2</a>, which requires you to download the assets (CSS, JavaScript) from GitHub
+
+<a id="option1"></a>
+
+# Option 1: Use NPM
 
 ## Requirements
+To use GOV.UK Frontend with NPM you must:
+
+1. Install the long-term support (LTS) version of [Node.js](https://nodejs.org/en/), which includes NPM.
+2. Create a [package.json file](https://docs.npmjs.com/files/package.json) if you don’t already have one. You can create a default package.json file by running `npm init` from the root of your application.
+
+## Installation
+GOV.UK Frontend is currently in private beta. You will need to log in to NPM using credentials provided by the Design System team.
+
+Run the following command to log in:
+`npm login`
```

Comment:
Code blocks like this need to use the fenced code block:

    ```
    Code in here
    ```

rather than the inline code block:

    `code in here`

So that the code is displayed as a block on it's own line:

```
Code in here
```

Rather than inline:
`code in here`

Reply:
Most of these could also be marked up with the relevant language (mostly js I think?) to enable syntax highlighting.

Reply:
Check with Alex - I'm not sure how to do this

Reply:
Done now